### PR TITLE
Energy Fire Axe

### PR DESCRIPTION
**Pull request by SapphicOverload**
# Document the changes in your pull request

Adds a new weapon to the uplink for 10 TC, available to nuclear operatives and traitors in the engineering department: the energy fire axe.

A stronger version of the fireaxe that when wielded, does the same damage as an energy sword, has the same block chance and slightly higher armor penetration (50 > 35), and deals twice as much damage to inanimate objects such as machines, structures and mechs. However unlike an energy sword it cannot fit in a pocket, box or chest rig while inactive and requires both hands to use, meaning you cannot use another item alongside it to further increase block chance unlike an energy sword. It can also be used as a lighter and will ignite anyone you hit that happens to be covered in flammable materials. Fits in the suit storage slot of syndicate hardsuits.

Uses the magnetic cleaver sprite from shiptest-ss13/Shiptest#464

# Spriting

Wielded:
image
image
image
image

One-handed:
image
image
image
image

Back/Storage:
image
image
image
image

Item:
image
image

# Wiki Documentation

If there's a list of stuff you can get in the uplink, this will need to be added to it.

# Changelog

:cl:  SapphicOverload, MemedHams
rscadd: adds energy fire axe
imageadd: adds energy fire axe sprites
tweak: adds "axed" to the list of attack verbs for axes
bugfix: fixes windows and grilles not respecting armor penetration
bugfix: holding an open flame will now ignite flammable gases in the air as intended
/:cl:

### CODE REVIEW

**Comment by isotope436** [comment]
why did you have to make it a nukie item instead of just making it a ruin spawn item thats a fancy syndi mag crusher like it actually is on tg 😢

File: code/game/objects/items/twohanded.dm [hunk 1]
```diff
@@ -288,6 +288,81 @@
...
+	sharpness = SHARP_NONE
+	resistance_flags = FIRE_PROOF | ACID_PROOF
+
+/obj/item/twohanded/fireaxe/energy/wield(mob/living/carbon/M) //Specific wield () to switch hitsounds.
+	..()
+	if(wielded)
+		sharpness = SHARP_EDGED
+		w_class = w_class_on
+		hitsound = 'sound/weapons/blade1.ogg'
+		START_PROCESSING(SSobj, src)
```
Comment (tattax) [inline code/game/objects/items/twohanded.dm:322]: ?

File: code/game/objects/items/twohanded.dm [hunk 2]
```diff
@@ -288,6 +288,81 @@
...
+/obj/item/twohanded/fireaxe/energy/attack(mob/living/M, mob/living/user)
+	..()
+	M.IgniteMob() // Ignites you if you're flammable
+
+/obj/item/twohanded/fireaxe/energy/afterattack(atom/A, mob/user, proximity)
+	. = ..()
+	if(!proximity)
+		return
+	if(wielded && !QDELETED(A)) // Does x2 damage against inanimate objects like machines, structures, mechs, etc
+		if(istype(A, /obj) && !istype(A, /obj/item))
```
Comment (tattax) [inline code/game/objects/items/twohanded.dm]:
```suggestion
		if(isobj(A) && !isitem(A))
```

**Comment by Cartlord** [comment]
Considering that the existing fire axe is quite strong and a tantalizing target for theft, I don't see the reason for a stronger version.

**Review by Moltijoe — APPROVED** [review]

**Comment by MajManatee** [comment]
i didn't realize it was an antag item, this is kinda za za'd
Shame it looks stupid when onehanded

**Review by cuackles — CHANGES_REQUESTED** [review]
image
think the hand is offset by one pixel, also the on back sprite strap has a bit of banding, could that be edited out so it looks more simple ?

**Comment by ChuChu42** [comment]
it increases the heat of the people hit by it
this isnt documented

**Comment by iamthesilly** [comment]
> it increases the heat of the people hit by it
> this isnt documented

No it doesn't, what part of the code are you looking at?

**Comment by iamthesilly** [comment]
> image think the hand is offset by one pixel, also the on back sprite strap has a bit of banding, could that be edited out so it looks more simple ?

image
is this better?

**Review by cuackles — APPROVED** [review]

**Pull request merged**